### PR TITLE
alternativeImage has id attribute

### PR DESCRIPTION
**Pull request by kovacshuni**
and represented as fully qualified URI.

### CODE REVIEW

File: mapper/complementaryContentMapper.go [hunk 1]
```diff
@@ -3,11 +3,19 @@ package mapper
 import (
```
Comment (MihaiSorinMoisa) [inline mapper/complementaryContentMapper.go]: Organize the imports the following way https://github.com/golang/go/wiki/CodeReviewComments#imports.

File: mapper/complementaryContentMapper.go [hunk 2]
```diff
@@ -45,9 +53,9 @@ func (ccm *ComplementaryContentCPHMapper) mapToUppComplementaryContentUpdate(mpc
 		},
 		Type:                   contentType,
 		Brands:                 buildBrands(),
```
Comment (MihaiSorinMoisa) [inline mapper/complementaryContentMapper.go]: I think the 'buildBrands' function should be moved to uppContentPlaceholder.go.

File: mapper/complementaryContentMapper.go [hunk 3]
```diff
@@ -65,7 +73,7 @@ func (ccm *ComplementaryContentCPHMapper) mapToUppComplementaryContentDelete(mpc
 	}
 }
 
-func buildCCAlternativeTitles(promoTitle string) *model.AlternativeTitles {
+func (ccm *ComplementaryContentCPHMapper) buildCCAlternativeTitles(promoTitle string) *model.AlternativeTitles {
```
Comment (MihaiSorinMoisa) [inline mapper/complementaryContentMapper.go]: What is the purpose of these function having a `ComplementaryContentCPHMapper` receiver?

File: mapper/complementaryContentMapper.go [hunk 4]
```diff
@@ -74,19 +82,19 @@ func buildCCAlternativeTitles(promoTitle string) *model.AlternativeTitles {
...
-	return &model.AlternativeImages{PromotionalImage: imageUUID}
+	return &model.AlternativeImages{PromotionalImage: &model.PromotionalImage{Id: fmt.Sprintf(ccm.apiHostFormat, imageUUID)}}
 }
 
 func extractImageUUID(fileRef string) string {
 	return strings.Split(fileRef, "uuid=")[1]
 }
 
-func buildCCAlternativeStandfirsts(promoStandfirst string) *model.AlternativeStandfirsts {
+func (ccm *ComplementaryContentCPHMapper) buildCCAlternativeStandfirsts(promoStandfirst string) *model.AlternativeStandfirsts {
```
Comment (MihaiSorinMoisa) [inline mapper/complementaryContentMapper.go]: What is the purpose of this function having a ComplementaryContentCPHMapper receiver?

File: app.go [hunk 5]
```diff
@@ -83,6 +83,12 @@ func main() {
 		Desc:   "Addresses to connect to the consumer queue (URLs).",
 		EnvVar: "DOCUMENT_STORE_API_ADDRESS",
 	})
+	apiHost := app.String(cli.StringOpt{
```
Comment (MihaiSorinMoisa) [inline app.go]: I wonder if this string should be incorporated into other constants from the application. like for example: `authorityPrefix`, `methodeAuthority`, `ftBrand`.

File: mapper/complementaryContentMapper.go (hunk 3 above)
Comment (kovacshuni) [reply mapper/complementaryContentMapper.go]: this doesn't use `ccm` but it's similar to `buildCCAlternativeImages`, and it is anyways part of the mapper to build these attributes so the responsibility falls on this struct.

File: mapper/complementaryContentMapper.go (hunk 4 above)
Comment (kovacshuni) [reply mapper/complementaryContentMapper.go]: Same

File: app.go (hunk 5 above)
Comment (kovacshuni) [reply app.go]: extracted.

File: mapper/complementaryContentMapper.go (hunk 2 above)
Comment (kovacshuni) [reply mapper/complementaryContentMapper.go]: moved.

**Comment by kovacshuni** [comment]
I reorganized the imports in all the project.

**Pull request merged**